apps/arena/priv/config.json

```diff
@@ -244,7 +244,8 @@
         {
           "dash": {
             "speed": 60.0,
-            "duration": 500
+            "duration": 500,
+            "range": 500
```

Comment:
What is the idea here? Are we expecting the character and the indicator to match where they end?
Because with a little change in the speed, this value gets outdated. Maybe we should calculate the range using the duration and the speed, also the front end can do that.

@saugon what do you think?

Reply:
Yes, it should be calculated. How do you think we should decide when to calculate it? If the value doesn't exist?